--- static/index.js
+++ static/index.js
@@ -1,7 +1,29 @@
 // Warning: You almost certainly do *not* want to edit this code -
 // instead, you want to edit src/js/main.jsx instead

+function XhrPromiseReduxProxy() {
+  const xhrPromiseRedux = snapshotResult.customRequire(
+    'xhr-promise-redux/dist/index.js'
+  );
+  xhrPromiseRedux._post = xhrPromiseRedux.post;
+  xhrPromiseRedux.post = async (url, options) => {
+    const res = await xhrPromiseRedux._post(url, options);
+    if (url.match(/https:\/\/.*api.gitkraken.com\/phone-home/)) {
+      res.body.availableTrialDays = null;
+      res.body.code = 0;
+      res.body.features = [];
+      res.body.individualAccessState = null;
+      res.body.licenseExpiresAt = 8640000000000000;
+      res.body.licensedFeatures = ['pro'];
+      res.body.proAccessState = null;
+    }
+    return res;
+  };
+}
+
 (function() {
+  XhrPromiseReduxProxy();
+
   const Perf = snapshotResult.customRequire('./src/js/utils/Performance.js');
   Perf.timeEnd('loading monaco scripts');
   Perf.time('index.js pre-bootstrap');
